bodo/tests/test_parquet_write.py

```diff
@@ -1038,6 +1078,42 @@ def impl(df, path):
         bodo.barrier()
 
 
+def test_to_pq_multiIdx(check_write_func, memory_leak_check):
+    """Test to_parquet with MultiIndexType"""
+    arrays = [
+        ["bar", "bar", "baz", "baz", "foo", "foo", "qux", "qux"],
+        ["one", "two", "one", "two", "one", "two", "one", "two"],
```

Comment:
Let's test arrays with different data types.

Reply:
Sure did so see above

Reply:
Updated the test